src/test/assembly/is_aligned.rs

```diff
@@ -33,7 +33,7 @@ pub fn is_aligned_1(ptr: *const u8) -> bool {
 // CHECK-LABEL: is_aligned_2
 // CHECK: testb $1
 // CHECK-NEXT: sete
-// CHECK-NEXT: retq
+// CHEC: retq
```

Comment:
that seems wrong 😆

Reply:
Oh, I didn't even see I deleted one letter too much lol